fzf-git.sh

```diff
@@ -256,8 +256,10 @@ elif [[ -n "${ZSH_VERSION:-}" ]]; then
     for o in "$@"; do
       eval "fzf-git-$o-widget() { local result=\$(_fzf_git_$o | __fzf_git_join); zle reset-prompt; LBUFFER+=\$result }"
       eval "zle -N fzf-git-$o-widget"
-      eval "bindkey '^g^${o[1]}' fzf-git-$o-widget"
-      eval "bindkey '^g${o[1]}' fzf-git-$o-widget"
+      for m in emacs vicmd viins; do
```

Comment:
```
__fzf_git_init:5: scalar parameter m created globally in function __fzf_git_init
```

Reply:
Regarding

> But why bind e.g., both ^Gf and ^G^F? 

I can explain that from a usability perspective. Depending on where you place your fingers and how fast you type, then both keystrokes are useful.

Of course, only one of them will respond and activate the action.

However, when typing fast, you can decide to use one hand to place a finger and press the CTRL 

At some point your finger will release CTRL.

Another finger or two can then press `G` followed by the action, for example `B`

You either release the CTRL finger before OR after pressing the action. This is why it is really smart to register both keyboard shortcuts 👍

Reply:
Thanks @jesperronn for the comment. Yes, that makes a lot of sense.

The real reason we decided to have two sets of bindings is described in https://github.com/junegunn/fzf-git.sh?tab=readme-ov-file#usage